Anti_Private.sp

```diff
@@ -278,55 +279,92 @@ void ParseProfile(const char[] sBody, int iClient)
 		return;
 		
 	int iProfile = json_object_get_int(hPlayer, "profilestate");
-	
+
 	if (iState == 3 && iProfile == 1)
 	{
 		if (!bInventory)
 			return;
-		
+			
+		char SteamID[64];
+	
+		GetClientAuthId(iClient, AuthId_SteamID64, SteamID, sizeof SteamID);
+
 		switch (GetEngineVersion())
 		{
 			case Engine_TF2, Engine_DOTA, Engine_Portal2: {}
+			case Engine_CSGO:
+				Format(InventoryURL,sizeof(InventoryURL),"https://steamcommunity.com/profiles/%s/inventory/json/730/2", SteamID);
 			default:
 				return;
 		}
-		
-		char SteamID[64];
-	
-		GetClientAuthId(iClient, AuthId_SteamID64, SteamID, sizeof SteamID);
-	
-		if (STEAMWORKS_AVAILABLE())
+		if(GetEngineVersion()==Engine_CSGO)

```

Comment:
Is there a reason why you are duplicating SteamWorks and SteamTools check just for CSGO.